Lib/test/test_lazy_imports.py

```diff
@@ -0,0 +1,17 @@
+import importlib
+import unittest
+
+class TestLazyImports(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(TestLazyImports, self).__init__(*args, **kwargs)
+
+    @unittest.skipIfLazyImportsIsDisabled("Skip this test when Lazy Import is disabled.")
```

Comment:
the "reason" is redundant with the decorator name

```suggestion
    @unittest.skipIfLazyImportsIsDisabled("Test relevant only when running with lazy imports enabled.")
```